database/data/role_list.json

```diff
@@ -0,0 +1 @@
+["Frontend", "Backend", "Fullstack", "Data Science"]
```

Comment:
En la daily he explicado que si un rol no tiene ningún rol asignado, en lugar de establecerlo como `null`, vamos a darle el valor `Pending to assign`. Ésto va a tener dos implicaciones:

1. Como no estamos usando `null`, nos ahorraremos las tipicas cláusulas que comprueban si el valor es nulo. Cuando eso ocurra iremos a buscar directamente los estados `Pending to assign`.
2.  Cuando vayamos a listar los roles, vamos a listar todos los estados excepto `Pending to assign`. Los alumnos con éste estado permanecerán ocultos siempre que el que lo esté viendo no tenga el permiso necesario para asignar al alumno un rol.

Reply:
¿Se puede hacer uso desde el BE de esta API para hacer track de todos los usuarios?
https://dev.sso.itawiki.eurecatacademy.org/api/v1/api-docs

Pertenece al plan de las apps de ITA, para evitar duplicidades de trabajo y usuarios en las mismas:
https://www.figma.com/file/DynJHHUlOiqx3F5h9dtvAW/Projectes-IT-Academy?type=design&node-id=2506-1747&mode=design&t=3tk3SE7yAJAgeD7j-0

Reply:
@kevinmamaqi de momento ésto es solo un prototipo. En futuras iteraciones trataremos la conexión con infraestructura, pero el tratamiento del dato pasará por nuestro dominio para adaptarlo al consumidor especifico de profiles.